src/components/header/icons/HistoryIcon.jsx

```diff
@@ -1,16 +1,18 @@
 import { Link } from "react-router-dom";
-import { useHistory } from "../../../context";
+import { useAuth, useHistory } from "../../../context";
 
 export function HistoryIcon() {
+  const { user } = useAuth();
   const { history } = useHistory();
+
   return (
     <Link to="/history" title="Go To History">
       <button className="sm-icon-btn color-dm sm-icon-btn-primary">
         <div className="icon">
           <i className="fas fa-history"></i>
           <span
```

Comment:
Suggestion: if you are using multiple icons, you could create an object consisting key as name of icon and value as html version of the icon.

For example :- 
`const icons = {
    history: <i className="fas fa-history"></i>,
    like:......,
    }
`

And then can be used as

`{icons.history}`